dotcom-rendering/src/components/AppsLightboxImageStore.importable.tsx

```diff
@@ -0,0 +1,56 @@
+import { Image } from '@guardian/bridget/Image';
+import { getGalleryClient } from '../lib/bridgetApi';
+import { generateImageURL } from '../lib/image';
+import type { ImageForAppsLightbox } from '../model/appsLightboxImages';
+
+// Gets assigned a value when the component is rendered
+let imageStore: ImageForAppsLightbox[];
```

Comment:
Should we handle the case when this is undefined?

```suggestion
// Gets assigned a value when the component is rendered
let imageStore: ImageForAppsLightbox[] | undefined;
```

Or alternatively an empty array to begin with?

```suggestion
// Gets assigned a value when the component is rendered
let imageStore: ImageForAppsLightbox[] = [];
```

Reply:
I think handling the `undefined` case makes sense! I'll add that.

Reply:
I ended up going with assigning an initial empty array.